doc/api.md

```diff
@@ -171,6 +171,69 @@ curl -s -w "%{http_code}" -X PATCH -H 'Content-Type: application/json'  -d '{
 
 ```
 
+#### **API:** DELETE  /horizondevice
+---
+
+Unconfigure the device so that it can be re-registered.
+
+**Parameters:**
+
+
+| name | type | description |
+| ---- | ---- | ---------------- |
+| block | bool | If true, the API blocks until the node is quiesced. If false, the caller will get control back quickly while the quiesce happens in the background. While this is occurring, the caller should call GET /horizondevice until they receive an HTTP status 404. The default is true. |
+| removeNode | bool | If true, the node’s entry in the currently configured exchange is also deleted, instead of just being sanitized. The default is false. |
+
+**Response:**
+
+code: 
+
+* 204 -- success
+
+body: 
+
+none
+
+**Example:**
+```
+curl -s -w "%{http_code}" -X DELETE -H 'Content-Type: application/json'  -d '{
```

Comment:
No, block and removeNode are not passed in the body, they are passed as query parameters.
curl -X DELETE http://localhost/horizondevice?block=true&removeNode=false